webpack.config.js

```diff
@@ -66,6 +66,54 @@ module.exports = function(env = {}) {
                     }
                     // tym razem tylko jeden loader
                 },
+                {
+                    test: /\.css$/,
+                //     // określam jakie pliki 
+                //     // będą brane pod uwagę
+                    exclude: /node_modules/,
+                //     // określam wykluczenia
+                    use: [
+                        isProd ? CssWebpackPlugin.loader : 'style-loader',  
+                    {
+                        loader: 'css-loader',
+                        options: {
+                            sourceMap: isProd ? false : true,
+                            // ustawiam identyfikację kodu źródłowego
+                        },
+                    },
+                ]
+                //     // określam jaki [loader]
+                //     // ma być wykorzystany
+                },
+                {
+                    test: /\.scss$/,
+                    // określam jakie pliki 
+                    // będą brane pod uwagę
+                    // exclude: /node_modules/,
+                    // określam wykluczenia
+                    use: [
+                        isProd ? CssWebpackPlugin.loader : 'style-loader', 
+                        // 'css-loader', 
+                        // 'sass-loader',
+                        { 
+                            loader: 'css-loader',
+                            options: {
+                                sourceMap: isProd ? false : true,
+                            }
+                        },
+                        {
+                            loader: 'sass-loader',
+                            options: {
+                                sourceMap: isProd ? false : true,
+                                sassOptions: {
+                                    outputStyle: isProd ? 'compressed' : 'expanded'
+                                }
+                            }
+                        }
+                    ]
+                    // określam jaki [loader]
+                    // ma być wykorzystany
+                },
```

Comment:
Zastanawiam się dlaczego jest to dodane. Jeśli korzystamy ze `StyledComponents` to nie musimy tego konfigurować.

Reply:
usunęłam